File: BasicsWithJava/BlockchainApp/app/src/main/java/com/sumere/blockchainapp/view/CryptoModel.java
package com.sumere.blockchainapp.view;

import com.google.gson.annotations.SerializedName;

public class CryptoModel {
    @SerializedName("currency")
    private String currency;
    @SerializedName("price")
    private String price;

    public CryptoModel(String currency, String price) {
        this.currency = currency;
        this.price = price;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }
}

File: CouroutinesKotlin/app/src/main/java/com/sumere/retrofitkotlin/model/CryptoModel.kt
package com.sumere.retrofitkotlin.model

import com.google.gson.annotations.SerializedName

data class CryptoModel(
    //Kotlin Feature do not need Serialized Name
    //data identifier already serializes in kotlin
    //@SerializedName("currency")
    val currency: String,
    //@SerializedName("price")
    val price: String) {
}